tartufo/scanner.py

```diff
@@ -49,13 +49,33 @@ def __init__(
         self.matched_string = matched_string
         self.chunk = chunk
 
-    def as_dict(self) -> Dict[str, Optional[str]]:
+    def _as_compact_dict(self) -> Dict[str, Optional[str]]:
+        """Return a compact dictionary representation of an issue.
+
+        This is primarily meant to aid in JSON serialization with fewer fields.
+
+        :return: A compact JSON serializable dictionary representation of this issue
+        """
+        return {
```

Comment:
This seems like code duplication to me. If the compact dict will always be a subset of the non-compact, I would prefer that the same code populates the fields in common.

Reply:
@jmink-godaddy Pretty sure I got this resolved for ya. :)